### PR TITLE
Update dynamo generation

### PR DESCRIPTION
**Pull request by btipling**
Removed names, prefixes and version.
pk and sk are assumed from field lists
only sk can use a const instead of a field list
version is generated from a field called updated_at that is a timestamp.
prefix is generated from message name.

### CODE REVIEW

File: internal/pgd/module.go [hunk 1]
```diff
@@ -241,53 +245,50 @@ func (m *Module) applyKeyFuncs(f *jen.File, in pgs.File) error {
...
-				name: "SortKey",
-			},
-			{
-				key:  mext.Version,
-				name: "Version",
-			},
-		}
+		keys := []namedKey{ }
+		// pk, sk, gsi1pk, gsi1sk
+		for _, ck := range mext.Key {
```
Comment (pquerna) [inline internal/pgd/module.go]: can this check for repeats?  eg re-using a GSI or PK?  or will that cause a code generatin failure elsewhere?

Comment (btipling) [reply internal/pgd/module.go]: Yes we should explicitly check it. Good call.

Comment (btipling) [reply internal/pgd/module.go]: @pquerna what do you think about getting rid id of the `global_secondary_index` and just increment. So you add multiple keys and the first is the pk, the second is the gsi1, etc.

Comment (btipling) [reply internal/pgd/module.go]: Removed in 3c5ac2d81b2fe6e30564e8499b3f7af1719502b0

File: internal/pgd/module.go [hunk 2]
```diff
@@ -316,21 +329,17 @@ func (m *Module) applyKeyFuncs(f *jen.File, in pgs.File) error {
...
-func generateKeyStringer(msg pgs.Message, stmts []jen.Code, ck *dynamopb.Key, stringBuffer string) []jen.Code {
+func generateKeyStringer(msg pgs.Message, stmts []jen.Code, fields []string, stringBuffer string) []jen.Code {
 	stmts = append(stmts, jen.Id(stringBuffer).Dot("Reset").Call())
 
-	sep := ck.Separator
-	if sep == "" {
-		sep = ":"
-	}
+	sep := ":"
+	prefix := msg.Name().LowerSnakeCase().String()
```
Comment (pquerna) [inline internal/pgd/module.go]: dont GSIs and the PK need a different initial prefix?  could just be `${prefix}_gsi1` i think for the gsi1 - its just to separate out the data from conflicts?

Comment (btipling) [reply internal/pgd/module.go]: gsi's are in a separate index no? Do they conflict?

Comment (btipling) [reply internal/pgd/module.go]: Based on my reading of

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SecondaryIndexes.html

It doesn't seem necessary, but I'll test it.

**Pull request merged**